Sources/StreamChatUI/CommonViews/BaseViews.swift

```diff
@@ -466,3 +466,7 @@ open class _TableViewCell: UITableViewCell, Customizable, AccessibilityView {
         super.layoutSubviews()
     }
 }
+
+/// Base class for overridable views StreamChatUI provides.
+/// All conformers will have StreamChatUI appearance settings by default.
+open class _TableViewHeaderFooterReusableView: _View {}
```

Comment:
Do we still need this? Since the view is actually not Reusable with the current implementation